### PR TITLE
Add port parameter to mysql DB load

### PR DESCRIPTION
**Pull request by edguy3**
As many do, I run mysql in docker on alternative ports, so the port parameter must be set in jest-mysql-config.
This change adds the -P <port> parameter to the mysql command line

### CODE REVIEW

**Comment by Daniel-Yonkov** [comment]
Thank you for your contribution @edguy3 
Would it be possible for you to introduce an unit test to cover the applied change in the config?
Dany

File: databaseSetup.js [hunk 1]
```diff
@@ -37,6 +37,9 @@ function importCreationScript(mysqlConfig, creationScriptPath) {
         if ("password" in mysqlConfig && mysqlConfig.password.length > 0) {
             params.push(`-p${mysqlConfig.password}`);
         }
+        if ("port" in mysqlConfig ) {
+            params.push(`-P ${mysqlConfig.port}`);
+        }
```
Comment (Daniel-Yonkov) [inline databaseSetup.js:42]: Please add unit test coverage for this new property

**Comment by edguy3** [comment]
> Thank you for your contribution @edguy3
> Would it be possible for you to introduce an unit test to cover the applied change in the config?
> Dany

Apologies.  Missed your message.   Will investigate shortly.

File: databaseSetup.js (hunk 1 above)
Comment (edguy3) [reply databaseSetup.js:42]: Latest change should suffice.

File: __tests__/environment/mysqlConfigParameters.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        callback(error, stdout, stderr);
+
+        execMethod = jest.fn();
+        execMethod(commandString);
+    });
+});
+
+
+it("Check DB config using Host & User only ", async () => {
+    const globalConfig = require("../../tests/configs/default.js");
```
Comment (Daniel-Yonkov) [inline __tests__/environment/mysqlConfigParameters.js:20]: No need for unused config loading
```suggestion
```

File: __tests__/environment/mysqlConfigParameters.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    };
+    const dummyPathToCreationScript = "FILE";
+    await expect(
+        importCreationScript(dummyMysqlConfig, dummyPathToCreationScript)
+    ).rejects.toBe("stderr");
+    expect(execMethod).toHaveBeenCalledWith('mysql -h 127.0.0.1 -u root  < FILE');
+});
+
+it("Check DB config using Port ", async () => {
+    const globalConfig = require("../../tests/configs/default.js");
```
Comment (Daniel-Yonkov) [inline __tests__/environment/mysqlConfigParameters.js:33]:
```suggestion
```

File: __tests__/environment/mysqlConfigParameters.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    const dummyPathToCreationScript = "FILE2";
+    await expect(
+        importCreationScript(dummyMysqlConfig, dummyPathToCreationScript)
+    ).rejects.toBe("stderr");
+    expect(execMethod).toHaveBeenCalledWith('mysql -h 127.0.0.1 -u root -P 3306  < FILE2');
+});
+
+
+it("Check DB config using password ", async () => {
+    const globalConfig = require("../../tests/configs/default.js");
```
Comment (Daniel-Yonkov) [inline __tests__/environment/mysqlConfigParameters.js:48]:
```suggestion
```

File: __tests__/environment/mysqlConfigParameters.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    const dummyPathToCreationScript = "FILE";
+    await expect(
+        importCreationScript(dummyMysqlConfig, dummyPathToCreationScript)
+    ).rejects.toBe("stderr");
+    expect(execMethod).toHaveBeenCalledWith('mysql -h 127.0.0.1 -u root -psuperSecret  < FILE');
+});
+
+
+it("Check DB config using database parameter ", async () => {
+    const globalConfig = require("../../tests/configs/default.js");
```
Comment (Daniel-Yonkov) [inline __tests__/environment/mysqlConfigParameters.js:63]:
```suggestion
```

File: __tests__/environment/mysqlConfigParameters.js [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    const dummyPathToCreationScript = "FILE";
+    await expect(
+        importCreationScript(dummyMysqlConfig, dummyPathToCreationScript)
+    ).rejects.toBe("stderr");
+    expect(execMethod).toHaveBeenCalledWith('mysql -h 127.0.0.1 -u root jest_mysql_test < FILE');
+});
+
+
+it("Check seven part config ", async () => {
+    const globalConfig = require("../../tests/configs/default.js");
```
Comment (Daniel-Yonkov) [inline __tests__/environment/mysqlConfigParameters.js:78]:
```suggestion
```

**Review by Daniel-Yonkov — CHANGES_REQUESTED** [review]
Found some unused imports. Could you remove them?
Thanks for the contribution, will merge it once ready with the minor changes suggested here

File: __tests__/environment/mysqlConfigParameters.js (hunk 4 above)
Comment (edguy3) [reply __tests__/environment/mysqlConfigParameters.js]: Done. Thanks for the additional eyes!

**Review by Daniel-Yonkov — APPROVED** [review]
Thanks for the edit. Will prepare the new release and will merge it into master

**Pull request merged**